### DISCUSSION TITLE
这个柱状图没有数据的时候就什么都不展示了

### DISCUSSION THREAD

**Discussion by zhiyou1201** [antvis/G2#5640] [Q&A]
![image](https://github.com/antvis/G2/assets/75651528/8d02a50e-10a8-4246-816c-c104c434a1d1)

这个柱状图没有数据的时候就什么都不展示了

![image](https://github.com/antvis/G2/assets/75651528/4ab546d1-5d38-43c2-9a1d-5cd91cf50062)

能不能没有数据的时候展示这样的 展示出来图的整体 没有数据也可以 把图的整体展示出来

![image](https://github.com/antvis/G2/assets/75651528/a6a470c7-9efa-4c46-9df3-c03651a875a6)

**Comment by pearmini** [comment]
当数据为 0 的时候，可以 mock 数据，然后把图形的 style.opacity 设置为 0，同时隐藏 y 轴的 tick

**Comment by zhiyou1201** [comment]
![image](https://github.com/antvis/G2/assets/75651528/a11d3b69-daa1-4bd3-8662-50dadec5b19e)
请问这个怎么解决  距离这么宽呢  能吧0放到x轴上吗

**Reply by pearmini** [reply]
y 轴必须是 number 类型

**Comment by zhiyou1201** [comment]
![image](https://github.com/antvis/G2/assets/75651528/795f9db4-3c86-45a4-996b-12a1d49fdb6c)

我是两组数据想展示到一起 不想要这样的

![image](https://github.com/antvis/G2/assets/75651528/5afaf6ab-3e59-46be-bbb1-35cc0b9f939a)

像要这样的

![image](https://github.com/antvis/G2/assets/75651528/65e634ed-41aa-4b87-8e0c-c7c4f1e066c6)

麻烦您帮我看看这个要怎么实现   谢谢您
